internal/cli/ci.go

```diff
@@ -0,0 +1,91 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+
+	"github.com/windmilleng/tilt/internal/analytics"
+	"github.com/windmilleng/tilt/internal/cloud"
+	"github.com/windmilleng/tilt/internal/engine/buildcontrol"
+	"github.com/windmilleng/tilt/internal/store"
+	"github.com/windmilleng/tilt/internal/tiltfile"
+	"github.com/windmilleng/tilt/pkg/logger"
+)
+
+type ciCmd struct {
+	fileName             string
+	outputSnapshotOnExit string
+}
+
+func (c *ciCmd) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "ci [<tilt flags>] [-- <Tiltfile args>]",
+		DisableFlagsInUseLine: true,
+		Short:                 "Start Tilt in CI/batch mode with the given Tiltfile args",
+		Long: `
+Starts Tilt and runs services defined in the Tiltfile.
+
+Exits with failure if any build fails or any server crashes.
```

Comment:
seems like the main difference between this and `--watch=false` (or whatever it's called now) is that this exits after pods are up and ready, and the latter exists after the final yaml apply is done. maybe call that out in the help text? (probably don't bother contrasting it to `--watch=false` but DO say when this command is "done".)

Reply:
For this particular documentation, I think we want to assume the user has never heard of --watch=false. Given that assumption, what do you hope to accomplish by introducing it here?

If we learn that `tilt ci` is a better fit for what most people need, then we don't want to encourage them to learn about --watch=false

Reply:
oh, nm, i misread what you were saying, added some more text on exit conditions